contracts/erc20/ERC20MintableBurnable.vy

```diff
@@ -68,17 +107,30 @@ def mint(_to: address, _value: uint256(wei)):
     self.balances[_to] = self.balances[_to] + _value
     log.Transfer(ZERO_ADDRESS, _to, _value)
 
+
+# @dev Internal function that burns an amount of the token of a given
+#      account.
+# @param _to The account whose tokens will be burned.
+# @param _value The amount that will be burned.
 @private
 def _burn(_to: address, _value: uint256(wei)):
     assert _to != ZERO_ADDRESS
     self.total_supply = self.total_supply - _value
     self.balances[_to] = self.balances[_to] - _value
     log.Transfer(_to, ZERO_ADDRESS, _value)
 
+
+# @dev Burn an amount of the token of a given account.
```

Comment:
`Burn an amount of the token`
or
`Burn an amount of the token of msg.sender` ?